src/plugins/core/java.rs

```diff
@@ -191,6 +204,21 @@ impl JavaPlugin {
                 tv.install_path().as_path(),
                 &tv.install_path().join("Contents").join("Home"),
             )?;
+        }
+
+        // if vendor is Zulu, symlink zulu-{major_version}.jdk/Contents to install path for macOS
+        if m.vendor.as_str() == "zulu" {
+            let major_version = m.version.split('.').next().unwrap();
```

Comment:
I think this would be a bit better:

```suggestion
            let (major_version, _) = m.version.split_once('.').unwrap_or_default(&m.version);
```

assuming it compiles that is

Reply:
I know they should always have a dot but unwraps make me uncomfortable and I've found if nothing else they're actually very noisy. If I'm diagnosing them I'm always drawn to `.unwrap()` even if it should always work